### PR TITLE
Economic CAB minutes for SIP-015

### PR DESCRIPTION
**Pull request by mattyTokenomics**
adding minutes for the Economic CAB discussion on SIP-015

### CODE REVIEW

File: considerations/minutes/economic-cab/2022-10-29-economic-cab-minutes-sip-15.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+## Meeting Notes
+
+- The change proposed in SIP-015 to count mining creates a *potential* economic risk and attack vector
+  - Specifically the change: "...the median Bitcoin spend of the miner's last six block-commits will include late block-commits."
+  - Depending on how this is implemented, a user could potentially 'spoof' having been *attempting* to mine a majority of the previous 6 blocks
+  - By selectively doing this only when mining happens to be profitable, or when the miners corresponding stacking slot is up for rewards miners can abuse system
+
+- The changes proposed in SIP-015 to stacking methods (namely enabling contintual renewal and increasing of stacking) are critical primitives
+  - As long as these changes are backwards compatible, this CAB sees no economic risks or externalities
+  - Next steps would be to enable liquid stacking by supporting the create of an 'xSTX' token, conceptually similar to 'stETH'. Each 1 xSTX token would represent 1 stacked STX toke that accrues stacking rewards
```
Comment (jcnelson) [inline considerations/minutes/economic-cab/2022-10-29-economic-cab-minutes-sip-15.md:36]: This exists: https://github.com/jcnelson/stx-future

Comment (mattyTokenomics) [reply considerations/minutes/economic-cab/2022-10-29-economic-cab-minutes-sip-15.md:36]: Thanks, will distribute it to rest of the Economics CAB, I think there may still be many edge cases and complexities in practically enabling liquid staked STX that need to be worked out (such as how to handle BTC rewards, the discrete cycle unlocks making a bit less fungible than stETH, etc)

File: considerations/minutes/economic-cab/2022-10-29-economic-cab-minutes-sip-15.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  - The Economics CAB was not formally required as a party to sign off on SIP-015
+  - Yet we found an *economic attack motivation* created by the SIP, that depending on *technical implementation* may or may not be viable
+  - The presence of such situations (in the very first CAB review no less) suggests that the overall SIP/CAB process should consider **all** CABs being required on SIPs, and those CABs required to either Approve, Disapprove, or formally indicate the SIP is not relevant to them
+  - This would make the process more burdensome, but given the high stakes and interdependencies involved in such complex systems, may be the wiser approach
+
+## Action Items
+
+- Raise the economic risk created in SIP-015 mentioned above to core SIP authors (Jude) [x]
+- Assemble an xSTX working group []
+- Confirm with the SIP process group if the CAB process itself should default require all CABs (i.e. opt-out, not opt-in) []
```
Comment (whoabuddy) [inline considerations/minutes/economic-cab/2022-10-29-economic-cab-minutes-sip-15.md:62]: Markdown supports basic task lists with the format `- [ ]` for an incomplete task and `- [x]` for a complete task, e.g.

- [ ] to do
- [x] done

Looks like you have it that way in the template :slightly_smiling_face: 

```suggestion
- [x] Raise the economic risk created in SIP-015 mentioned above to core SIP authors (Jude)
- [ ] Assemble an xSTX working group
- [ ] Confirm with the SIP process group if the CAB process itself should default require all CABs (i.e. opt-out, not opt-in)
```

File: considerations/minutes/economic-cab/2022-10-29-economic-cab-minutes-sip-15.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Meeting Information
+
+**Location:** Google Meet
+
+**Materials**: [Sip-015 proposal](https://github.com/stacksgov/sips/blob/feat/sip-015/sips/sip-015/sip-015-network-upgrade.md)
+
+**Date:** Oct 29 2022
+
+**Time:** 1pm ET
```
Comment (whoabuddy) [inline considerations/minutes/economic-cab/2022-10-29-economic-cab-minutes-sip-15.md:11]: Can you put this in UTC?

**Review by whoabuddy — CHANGES_REQUESTED** [review]
Minor formatting stuff but otherwise LGTM.

@jcnelson or @mattyTokenomics any reason not to merge it after the changes?

File: considerations/minutes/economic-cab/2022-10-29-economic-cab-minutes-sip-15.md (hunk 3 above)
Comment (whoabuddy) [reply considerations/minutes/economic-cab/2022-10-29-economic-cab-minutes-sip-15.md:11]: Did the conversion, this should be the update below assuming the meeting was scheduled for 1hr:

**Time:** 17:00 - 18:00 UTC

**Comment by whoabuddy** [comment]
@mattyTokenomics if all looks good here re: suggestions happy to get the final changes done so we can merge it in.

I also like how you added the template file - once all three CABs have their minutes in we can loop back and generalize the format for everyone - great idea!

**Comment by mattyTokenomics** [comment]
requested changes made

**Review by whoabuddy — APPROVED** [review]

**Comment by jcnelson** [comment]
Just wanted to let the economics CAB know that their main concern with SIP-015 is addressed in this PR, if they want to take a look: https://github.com/stacks-network/stacks-blockchain/pull/3379

**Pull request merged**